rental.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'book'
+require_relative 'person'
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
```

Comment:
- [ ] You should add the rental itself to the @person the 'belongs-to' association, Please check that hint 👇

```ruby
    person.rentals << self
```